dts/bindings/bluetooth/zephyr,bt-hci-ipc.yaml

```diff
@@ -0,0 +1,17 @@
+description: Bluetooth HCI using the IPC subsystem
+
+compatible: "zephyr,bt-hci-ipc"
+
+include: bt-hci.yaml
+
+properties:
+  bt-hci-name:
+    default: "IPC"
+  bt-hci-bus:
+    default: "BT_HCI_BUS_IPM"
+  bt-hci-quirks:
+    default: ["BT_HCI_QUIRK_NO_AUTO_DLE"]
+  bt-hci-ipc-name:
+    type: string
+    default: "nrf_bt_hci"
+    description: IPC endpoint name
```

Comment:
Note: the default "nrf_bt_hci" string is probably not the best one, since there's nothing nrf-specific about this. What's important is that this matches what the controller side uses, i.e. in most cases `samples/bluetooth/hci_ipc`:
https://github.com/zephyrproject-rtos/zephyr/blob/53ddff639562ef68dc0a6f62b82f7505c75ebdce/samples/bluetooth/hci_ipc/src/main.c#L369-L370

On the app-side it might be better to have this as an app-specific Kconfig option.

@kapi-no FYI

Reply:
> It's based on openamp, right? Maybe that can replace the nRF part somehow? Or am I misunderstanding this comment?

I just meant that we could remove "nrf_" from the string. Unltimately all that matters is that both sides of the IPC use the same string since otherwise they will not discover each other.